peppy/project.py

```diff
@@ -441,20 +441,67 @@ def templates_folder(self):
         return os.path.join(os.path.dirname(__file__), "submit_templates")
 
 
-    @staticmethod
```

Comment:
If this is going to be used for effect and no longer be static, the logic should live directly in the constructor rather than in this function.

Reply:
does it seem more natural to override a function in a child class?

Reply:
Ahhhh I see! Hadn't thought about overrides; that definitely makes more sense if the idea is to encourage or at least facilitate overriding.

Reply:
yeah -- originally you were overriding this in the `Project` child in `looper`.